lib/reek/cli/options.rb

```diff
@@ -13,9 +13,9 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 12 }
-    # :reek:TooManyMethods: { max_methods: 18 }
-    # :reek:Attribute: { enabled: false }
```

Comment:
I missed those occurences and about 20 others in #1308 big time.

Reply:
How come these don't show up as smells when running reek on itself? :confused:

Reply:
Ah! It just ignores everything after the third `:`.